cfgov/jinja2/v1/_includes/organisms/chart.html

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
This mostly taken from @cfarm 's pull request https://github.com/cfpb/cfgov-refresh/pull/2656